### DISCUSSION TITLE
time need some love, so I will rewrite it.

### DISCUSSION THREAD

**Discussion by enghitalo** [vlang/v#20908] [Blog]
### proposal 
- [ ] Change `Time` struct (softly, deprecating old fields in the proccess)
```v
pub struct Time {
pub:
	year	                       		int
	month                      		int
	day                        		int
	hour                       		int
	minute                     		int
	second                     		int
	nanosecond                 		int
	unix                       		i64
	offset_from_utc_in_minutes 		int // values between `- 12 * 60` = -720 until `12 * 60` = 720
	daylight_saving_time_in_minutes 	int
	precision_in_nanosecond			int
}
```
- [ ] change `parse` to submodule
- [ ] change `format` to submodule ?
- [ ] create `stopwatch` submodule
- [ ] reduce or remove the use of `Duration` Alias or put it in a submodule
- [ ] use `strftime` syntax as default for formating (I need to do some benchmark to verify if it is worth it) 
- [ ] speedup time through use of proper clock id anchor to precision field
- [ ] pre-compile localtime information

- Discord discussion (https://discord.com/channels/592103645835821068/592106336838352923/1211502960140484678)

**Comment by spytheman** [comment]
What are your goals/motivations for the points above?
